changes/5859.housekeeping

```diff
@@ -0,0 +1,2 @@
+Changed `--cache-test-fixtures` and `--keepdb` flags from opt-in to opt-out for `inv unittest` and `inv integration-test` commands.
```

Comment:
Do any of our development/testing docs need to be updated to describe this and explain when you might want to opt out?